source/sass/blocks/programs.scss

```
@@ -1,24 +1,78 @@
 .programs {
   display: grid;
   row-gap: 20px;
+
+  @media (min-width: $tablet-width) {
+    justify-content: space-between;
+    grid-template-columns: 1fr 1fr 1fr 1fr;
```

Кажется что так удобнее)
```suggestion
    grid-template-columns: repeat(4, 1fr);
```